api/provider/aws/service_create.go

```diff
@@ -53,10 +51,20 @@ func (s *ServiceProvider) Create(req models.CreateServiceRequest) (string, error
 				}
 			}
 		}
+	}
 
+	scale := req.Scale
+	if req.Scale == 0 {
```

Comment:
What if I wanted the scale to be 0 initially?

Reply:
makes more sense to me to just now allow that.

Reply:
Not sure what you mean? Makes more sense to allow 0? or just now allow a scale of 0?

Reply:
Just a guess but "now" seems like a typo... should be "not". What's a practical reason to specify a scale of 0 initially? Can't think of any off the top of my head...